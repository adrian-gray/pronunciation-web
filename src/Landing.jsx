import React from 'react'

export default function Landing () {
  return (
    <div className='container'>
      <h1>This is landing page</h1>
      <hr />
    </div>
  )
}
