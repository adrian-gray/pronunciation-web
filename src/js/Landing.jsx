import React from 'react'
import data from './../data'

// import { splitHilite } from './Utils'

// import CommonWords from './components/CommonWords'
// import Description from './components/Description'
// import Dialogues from './components/Dialogues'
// import FindTheWords from './components/FindTheWords'
// import HearTheWords from './components/HearTheWords'
// import IdentifySounds from './components/IdentifySounds'
// import OddOneOut from './components/OddOneOut'
// import Phrases from './components/Phrases'
// import SaySentences from './components/SaySentences'
// import SoundTitle from './components/SoundTitle'
// import TongueTwisters from './components/TongueTwisters'
// import Words from './components/Words'
import WordsMaze from './components/WordsMaze'

export default function Landing () {
  return (
    <div className='container'>
      {/*

      <SoundTitle phoneme={data.phoneme} str={data.title} />
      <hr />

      <img src='/images/animation.png' />

      <Description phoneme={data.phoneme} arr={data.text} />
      <hr />

      <img src='/images/video1.png' />

      <Words words={data.words} />
      <hr />

      <Phrases words={data.phrases} />

      <hr />

      <Dialogues dialogues={data.dialogues} />

      <img src='/images/dialogues.png' />
      <hr />

      <h3>{'Common Words'}</h3>

      <CommonWords words={data.commonWords} />

      <hr />
      <h2>Member section</h2>
      <br />

      <h3>Tongue twisters</h3>
      <TongueTwisters />
      <hr />

      <h3>Find the words</h3>
      <h4>Click (✓) the words with the ‘short a’ <span className='em'>/æ/</span> sound</h4>
      <FindTheWords words={data.findTheWords} />
      <hr />

      <h3>Hear the words</h3>
      <h4>Underline the ‘short a’ /æ/ sound you hear</h4>
      <HearTheWords words={data.hearTheWords} />

      <hr />

      <h3><em>Say</em> the sentences below</h3>
      <SaySentences sentences={data.saySentences} />

      <hr />

      <h3>The odd one out</h3>
      <h4>Circle the words that don’t have the ‘short a’ /æ/ sound</h4>
      <OddOneOut words={data.oddOneOut} />

      <hr />

      <h3>Identify the sounds</h3>
      <h5>{splitHilite('Select the correct sound: ~/æ/~ (the short ‘a’ sound) or ~/eɪ/~ (the long ‘a’ sound)')}</h5>
      <h5>Snake Eats Tongs</h5>
      <IdentifySounds sentences={data.identifySounds} />

      <hr /> */}

      <h3>Words Maze</h3>
      <WordsMaze words={data.wordsMaze.words} correct={data.wordsMaze.correct} />
    </div>
  )
}
