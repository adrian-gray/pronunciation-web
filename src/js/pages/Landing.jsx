import React from 'react'

export default function Landing () {
  return (
    <div className='container'>
      <h1>Hello World</h1>
    </div>
  )
}
