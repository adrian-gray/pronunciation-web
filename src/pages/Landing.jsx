import React from "react";
import { Link } from "react-router-dom";

import Button from "@material-ui/core/Button";
import Paper from "@material-ui/core/Paper";
import Typography from "@material-ui/core/Typography";
import withStyles from "@material-ui/styles/withStyles";

import Mailchimp from "./../components/Mailchimp";
import SEO from "./../components/SEO";

const styles = theme => ({
  button: {
    marginTop: theme.spacing.unit,
    marginBottom: 2 * theme.spacing.unit,
    marginLeft: "auto",
    marginRight: "auto"
  },
  page: theme.page,
  headspace: theme.headspace,
  para: {
    paddingTop: "0.5em",
    paddingBottom: "0.5em"
  },
  video: {
    width: "100%"
  },
  startBtn: {
    textAlign: "center",
    padding: "2em"
  }
});

const Landing = props => {
  const { classes } = props;

  return (
    <Paper className={classes.page}>
      <SEO meta="landing" />
      <Typography variant="display1" gutterBottom>
        {"Pronunciation Practice"}
      </Typography>
      <Typography variant="title" gutterBottom>
        {"Improve Your Spoken English"}
      </Typography>
      <Typography className={classes.para} gutterBottom>
        {
          "Is poor English pronunciation holding you back in your career or studies? Do you want to improve your spoken English, but are not sure how? Help is on the way!"
        }
      </Typography>
      <Typography className={classes.para} gutterBottom>
        {
          "We are currently producing over 60 videos lessons to help you improve your spoken English. The videos feature Nicky Brooks, a professional English language consultant, giving English language pronunciation lessons made for this site. She teaches how to make the sounds used in English, how to understand English sounds, and how to improve your accent in English."
        }
      </Typography>
      <Typography className={classes.para} gutterBottom>
        {
          "Following the pronunciation lessons are English pronunciation examples, recorded conversations, and interactive activities to help improve your pronunciation the right way. We are adding currently adding sounds and activities weekly. Improve your English pronunciation to speak clearly and professionally."
        }
      </Typography>
      <Typography variant="title" gutterBottom>
        {"Stay up to date as we add pronunciation activities"}
      </Typography>
      <Typography className={classes.headspace} gutterBottom>
        {
          "We are currently adding sounds and activities multiple times a week and are open in a limited testing capacity. We aim to launch in late 2018."
        }
      </Typography>
      <Mailchimp />
      <div className={classes.startBtn}>
        <Button
          variant="contained"
          size="large"
          className={classes.button}
          component={Link}
          to="/home"
        >
          {"Start Pronunciation Practice"}
        </Button>
      </div>
      <img
        className={classes.video}
        src="/assets/images/improve-english-pronounciation.jpg"
        title="Learn to improve English pronunciation"
        alt="Learn to improve English pronunciation"
      />
    </Paper>
  );
};

export default withStyles(styles)(Landing);
