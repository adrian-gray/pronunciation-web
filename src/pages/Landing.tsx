import React from "react";
import { Link } from "react-router-dom";

import Button from "react-bootstrap/Button";
import Container from "react-bootstrap/Container";

import Mailchimp from "../components/Mailchimp";
import SEO from "../components/SEO";

export default () => {
  return (
    <Container>
      <SEO meta="landing" />
      <h1>{"Pronunciation Practice"}</h1>
      <h3>{"Improve Your Spoken English"}</h3>
      <p>
        {
          "Is poor English pronunciation holding you back in your career or studies? Do you want to improve your spoken English, but are not sure how? Help is on the way!"
        }
      </p>
      <p>
        {
          "We are currently producing over 60 videos lessons to help you improve your spoken English. The videos feature Nicky Brooks, a professional English language consultant, giving English language pronunciation lessons made for this site. She teaches how to make the sounds used in English, how to understand English sounds, and how to improve your accent in English."
        }
      </p>
      <p>
        {
          "Following the pronunciation lessons are English pronunciation examples, recorded conversations, and interactive activities to help improve your pronunciation the right way. We are adding currently adding sounds and activities weekly. Improve your English pronunciation to speak clearly and professionally."
        }
      </p>
      <h5>{"Stay up to date as we add pronunciation activities"}</h5>
      <p>
        {
          "We are currently adding sounds and activities multiple times a week and are open in a limited testing capacity. We aim to launch in late 2019."
        }
      </p>
      <Mailchimp />
      <div className="text-center full-width pad2">
        <Link to="/home">
          <Button variant="success" className="btn btn-lg">
            {"Start Pronunciation Practice Now"}
          </Button>
        </Link>
      </div>
      <img
        className="full-width"
        src="/assets/images/improve-english-pronounciation.jpg"
        title="Learn to improve English pronunciation"
        alt="Learn to improve English pronunciation"
      />
    </Container>
  );
};
