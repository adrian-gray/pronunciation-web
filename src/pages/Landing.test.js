import React from "react";
import { BrowserRouter } from "react-router-dom";

import { render, cleanup } from "@testing-library/react";

import Landing from "./Landing";

afterEach(cleanup);

test("Landing", async () => {
  const props = {};

  const { container } = render(
    <BrowserRouter>
      <Landing {...props} />
    </BrowserRouter>
  );

  expect(container).toMatchSnapshot();
});
