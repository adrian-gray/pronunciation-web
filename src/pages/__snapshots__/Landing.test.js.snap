// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing 1`] = `
<div>
  <div
    class="page"
  >
    <h1>
      Pronunciation Practice
    </h1>
    <h3>
      Improve Your Spoken English
    </h3>
    <p>
      Is poor English pronunciation holding you back in your career or studies? Do you want to improve your spoken English, but are not sure how? Help is on the way!
    </p>
    <p>
      We are currently producing over 60 videos lessons to help you improve your spoken English. The videos feature Nicky Brooks, a professional English language consultant, giving English language pronunciation lessons made for this site. She teaches how to make the sounds used in English, how to understand English sounds, and how to improve your accent in English.
    </p>
    <p>
      Following the pronunciation lessons are English pronunciation examples, recorded conversations, and interactive activities to help improve your pronunciation the right way. We are currently adding sounds and activities weekly. Improve your English pronunciation to speak clearly and professionally.
    </p>
    <h5>
      Stay up to date as we add pronunciation activities
    </h5>
    <p>
      We are currently adding sounds and activities weekly and are open in a limited testing capacity. We aim to launch in early 2020.
    </p>
    <div
      class="sub-canvas mailchimp-bg"
    >
      <div
        class="mailchimp-subscribe"
      >
        <div
          class="mailchimp-root"
        >
          <form
            class=""
          >
            <div
              class="form-row"
            >
              <label
                class="form-label"
              >
                Your first name
              </label>
              <input
                autocomplete="name"
                class="form-control form-bottom-space form-control"
                id="signin_email"
                placeholder="Your first name"
                value=""
              />
            </div>
            <div
              class="form-row"
            >
              <label
                class="form-label"
              >
                Your Email
              </label>
              <input
                autocomplete="email"
                class="form-control form-bottom-space form-control"
                id="email"
                placeholder="Your Email"
                value=""
              />
            </div>
            <div
              class="form-row"
            >
              <br />
              <p
                class="form-divider"
              >
                We hate spam and will never share your details with anyone else.
              </p>
              <br />
              <button
                class="btn btn-warning"
                type="button"
              >
                Keep me updated!
              </button>
            </div>
          </form>
          <br />
        </div>
      </div>
    </div>
    <div
      class="text-center full-width pad2"
    >
      <a
        href="/home"
      >
        <button
          class="btn btn-lg btn btn-success"
          type="button"
        >
          Start Pronunciation Practice Now
        </button>
      </a>
    </div>
    <div
      class="video-container"
    >
      <iframe
        allowfullscreen=""
        data-allow="autoplay; encrypted-media"
        data-type="text/html"
        frameborder="0"
        height="480"
        src="https://www.youtube.com/embed/eisQ1bONHP4"
        title="Improve English pronunciation"
        width="853"
      />
    </div>
  </div>
</div>
`;
